INSS.EIIR.Interfaces/INSS.EIIR.Interfaces.csproj

```diff
@@ -5,15 +5,12 @@
     <ImplicitUsings>enable</ImplicitUsings>
   </PropertyGroup>
 
-  <ItemGroup>
-    <Folder Include="DataAccess\" />
-  </ItemGroup>
-
   <ItemGroup>
     <PackageReference Include="Microsoft.Extensions.Logging.Abstractions" Version="6.0.0" />
   </ItemGroup>
 
   <ItemGroup>
+    <ProjectReference Include="..\INSS.EIIR.Data\INSS.EIIR.Data.csproj" />
```

Comment:
why do we have this reference? EF objects should not escape the dataaccess layer. Feels like a mapping to domain object missing

Reply:
This didn't feel right but I couldn't see how this was avoided will take another look. Typically I would expect an automapper implementation or alike.